ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py

```diff
@@ -2081,3 +2084,37 @@ def test_torch_sigmoid(
         on_device=on_device,
         input=x[0],
     )
+
+
+# lerp
+@handle_frontend_test(
+    fn_tree="torch.lerp",
+    dtype_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        default=0.5,
+    ),
+)
+def test_torch_lerp(
+    *,
+    dtype_and_input,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    input_dtype, inputs = dtype_and_input
+    start, end, weight = inputs
+
+    expected_output = torch.lerp(start, end, weight)
+    output = frontend.torch.lerp(start, end, weight)
+
+    assert_allclose(expected_output, output, rtol=1e-03, atol=1e-05)
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input=(start, end, weight),
+        expected_output=expected_output,
```

Comment:
don't pack all input arguments into one, make them seperate keyword args, i.e. `input=start`, `end=end`, `weight=weight`.
Remove `expected_output`
No need to include the `out` argument when you add it